Makefile

```diff
@@ -24,6 +24,26 @@ all build:
 	hack/build-go.sh
 .PHONY: all build
 
+
+# Verify code is properly organized.
+#
+# Example:
+#   make verify
+verify: build
+	hack/verify-gofmt.sh
+	hack/verify-golint.sh || true
+	hack/verify-govet.sh || true
```

Comment:
why || true?  what's the point of the verify target if it doesn't fail when things don't verify?